src/components/layouts/SearchBar.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import Autosuggest from "react-autosuggest";
+
+
+const organizations = [
+    {
+        name: 'Heartpoints',
+        url: 'http://hearpoints.org'
+    },
+    {
+        name: 'Some Organization',
+        url: 'http://google.com'
+    },
+    {
+        name: 'Altruistic Company',
+        url: 'http://bing.com'
+    }
+];
+
+export const SearchBar = (props) => {
+    const { searchBarValue } = props;
+    console.log({searchBarValue})
+
+    const getSuggestions = (searchBarValue) => {
+        const inputValue = searchBarValue.trim().toLowerCase();
+        const inputLength = inputValue.length;
+
+        return inputLength === 0 ? [] : organizations.filter(org =>
+            org.name.toLowerCase().slice(0, inputLength) === inputValue
```

Comment:
can we instead filter by the org name containing the search term instead of starting with it?

Reply:
This is a work in progress PR. Sorry for not specifying. I'm just including pieces as I come to understand them. Just had a grand old time with the css.

I will definitely look in to figuring out how to search the organizations that /contain/ the search term, not just begin with it.

Reply:
Yeah, when the app refreshes it seems to pull autofocus to the Facebook module. I'm trying to figure out that bug.

Also, sorry for the late replies. Apparently when i set up Github to notify me, it didn't work out.

Reply:
test